langchain/llms/mlflow.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from typing import Any, List, Dict, Mapping
+from langchain.callbacks.manager import CallbackManagerForLLMRun
+from langchain.llms.base import LLM
+import mlflow.gateway
+
+
+class MlflowGateway(LLM):
+    gateway_uri: str
+    route: str
+    temperature: float = 0.3
```

Comment:
We'll need to support additional standard parameters from https://docs.google.com/document/d/1AEDlqSWh4n0j1Ue-llk03aRz7-xBK4P4Zh-Yvo1uhnI/edit like max_tokens, right?

Reply:
We should also adjust the default value to be 0

Reply:
I'll add them and fix the default